templates/python-common/bin/check_review_threads.py

```
@@ -46,6 +51,37 @@
+# It is the worst possible failure for a gate: permanently, silently green. Both sides are now
+# normalised, and `test_review_threads_gate.py` pins the bot-suffix case by name.
+_BOT_SUFFIX = "[bot]"
+
+
+def normalise_login(str_login: str) -> str:
+	"""Return a login comparable across GitHub's REST and GraphQL spellings.
+
+	Parameters
+	----------
+	str_login : str
+	    A login as either API reports it, e.g. ``coderabbitai[bot]`` or ``coderabbitai``.
```

Ruff (0.16.1)

[error] Indentation contains mixed spaces and tabs

(E101)